tools/kagami/src/main.rs

```diff
@@ -36,6 +36,7 @@ pub trait RunArgs<T: Write> {
 /// Tool generating the cryptographic key pairs, schema, genesis block and configuration reference.
 #[derive(StructOpt, Debug)]
 #[structopt(name = "kagami", version, author)]
+#[allow(variant_size_differences)]
```

Comment:
I wouldn't allow this. This needs to be fixed with boxing, and not hidden.

Reply:
I figured it doesn't make a lot of difference when it's instantiated only once, but I can box offending variants instead, sure.